doc/rados/operations/placement-groups.rst

```diff
@@ -433,9 +433,11 @@ You should then check if the result makes sense with the way you
 designed your Ceph cluster to maximize `data durability`_,
 `object distribution`_ and minimize `resource usage`_.
 
-The result should be **rounded up to the nearest power of two.**
-Rounding up is optional, but recommended for CRUSH to more evenly balance
-the number of objects among placement groups.
+The result should always be **rounded up to the nearest power of two.**
+The way the CRUSH algorithm is designed, a power of two leads to a more 
```

Comment:
nit: it's actually rados, not crush, that figures out how PGs shard.  Maybe just ``A power of two ...``